src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,31 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { getUser } from '../../api';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
 import { Loader } from '../Loader';
 
-export const TodoModal: React.FC = () => {
+type Props = {
+  userId: number;
+  todo: Todo;
+  reset: () => void;
+};
+
+export const TodoModal: React.FC<Props> = ({ userId, todo, reset }) => {
+  const [user, setUser] = useState<User | null>(null);
+
+  useEffect(() => {
+    getUser(userId)
+      .then(setUser)
+      .catch(error => {
+        throw new Error(error);
```

Comment:
I suggest to show this error on UI

Reply:
not fixed
![image](https://user-images.githubusercontent.com/98764641/221129740-fec2f3e3-52e6-41a8-901a-16897f3ee79b.png)
would be better to show a notification to the user. he can't see the error